options/locale/locale_en-US.ini

```diff
@@ -83,21 +83,21 @@ general_title = General Application Settings
 app_name = Application Name
 app_name_helper = You can put your organization name here.
 repo_path = Repository Root Path
-repo_path_helper = All remote Git repositories will be saved to this directory.
+repo_path_helper = All Git repositories will be saved to this directory.
 lfs_path = LFS Root Path
 lfs_path_helper = Files stored with Git LFS will be stored in this directory. Leave empty to disable LFS.
 run_user = Run User
 run_user_helper = The user must have access to Repository Root Path and run Gitea.
-domain = Domain
-domain_helper = This affects SSH clone URLs.
+domain = SSH Domain
+domain_helper = This domain name is used to construct SSH clone URLs.
 ssh_port = SSH Port
 ssh_port_helper = Port number which your SSH server is using. Leave it empty to disable.
-http_port = HTTP Port
-http_port_helper = Port number which application will listen on.
+http_port = Local HTTP Port
+http_port_helper = The port on which gitea serves the API in its execution context (e.g. a container).
```

Comment:
It's not only API but all http requests, so instead of `API` it would probably be `web`

Reply:
maybe there's a misunderstand, what is this value used for other than calls to the api by igitea itself? at least that was my conclusion when i realised i just had to leave it as `3000` when running from the provided Docker image while that port is mapped to another one externally.

Reply:
It is the port gitea web will listen on for web/API/git http requests (localy) and http proxy in front needs to forward requests to this port if used.

Reply:
a'ight, amended.

Reply:
There should be space between web and interface